cmd/node/factory/structs.go

```diff
@@ -2042,6 +2049,7 @@ func newMetaBlockProcessor(
 	stateCheckpointModulus uint,
 	messageSignVerifier vm.MessageSignVerifier,
 	gasSchedule map[string]map[string]uint64,
+	txTypeHandler process.TxTypeHandler,
```

Comment:
no need to pass txtype handler. You can create it directly in newMetaBlockProcessor method and newShardBlockProcessor.code as it was before.

Reply:
done.